articles/node_lifecycle.md

```diff
@@ -27,224 +27,447 @@ Last Modified: {% if page.last_modified %}{{ page.last_modified }}{% else %}{{ p
+Therefor it is expected that user code catches errors and exceptions, returning `CallbackReturn::ERROR` or calling `raise_error()` when appropriate (e.g., in the `catch` or equivalent statement).
 
 ## Management Interface
 
-A managed node will be exposed to the ROS ecosystem by the following interface, as seen by tools that perform the managing.
+A managed node will be exposed to the `ROS 2` ecosystem by the following interface, as seen by tools that perform the managing.
 This interface should not be subject to the restrictions on communications imposed by the lifecycle states.
 
 It is expected that a common pattern will be to have a container class which loads a managed node implementation from a library and through a plugin architecture automatically exposes the required management interface via methods and the container is not subject to the lifecycle management.
 However, it is fully valid to consider any implementation which provides this interface and follows the lifecycle policies a managed node.
-Conversely, any object that provides these services but does not behave in the way defined in the life cycle state machine is malformed.
+Conversely, any object that provides these services but does not behave in the way defined in the Lifecycle state machine is malformed.
 
-These services may also be provided via attributes and method calls (for local management) in addition to being exposed ROS messages and topics/services (for remote management).
-In the case of providing a ROS middleware interface, specific topics must be used, and they should be placed in a suitable namespace.
+These services may also be provided via attributes and method calls (for local management) in addition to being exposed `ROS` messages and topics/services (for remote management).
+In the case of providing a `ROS` middleware interface, specific topics must be used, and they should be placed in a suitable namespace.
 
 Each possible supervisory transition will be provided as a service by the name of the transition except `create`.
 `create` will require an extra argument for finding the node to instantiate.
 The service will report whether the transition was successfully completed.
 
-### Lifecycle events
-
-A topic should be provided to broadcast the new life cycle state when it changes.
-This topic must be latched.
-The topic must be named `lifecycle_state` it will carry both the end state and the transition, with result code.
-It will publish every time that a transition is triggered, whether successful or not.
+### Supported Interfaces
+
+This section outlines Persistent Entities created by the backend of Lifecycle nodes. These Persistent Entities make up the external facing management interface of a Lifecycle node. Each of these external interfaces must also include an internal equivalent (i.e., a function the user can call from within the node itself).
```

Comment:
L373 - L383 I have no idea what you're trying to communicate or why its an important thing to distinctly discuss

Reply:
Actually, this entire subsection "Supported Interfaces" I don't understand. Something is really unclear about what the intention is. There's some stuff about the ROS interfaces (e.g. topics/services) which I ack as standard practice, but no idea what anything before "The following interface behaviors will be supported" seems unrelated. 

Also, all the interfaces are CamelCase and if these are supposed to be topics, then theey should be snake_case. If these are interface message types, that's unclear.